addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -590,6 +582,18 @@ export class LinkPlugin extends Plugin {
         }
     }
 
+    checkUrlTextAligned() {
```

Comment:
By "aligned", you mean "was the same"? It's not wrong, but when I see align in the context of the editor I think about "left, right, center, justify", first... ^^' I suggest to rename the function to `updateCurrentLinkSyncState`, and the variable to `this.isCurrentLinkSynced` or `this.isCurrentLinkInSync`.

Reply:
I will change it, thanks! Not good at making names :rofl: